docs/site/getting-started.md

```diff
@@ -60,6 +60,31 @@ Open `app.config.ts` and register SCION Workbench providers. Added lines are mar
 ```
 </details>
 
+<details>
```

Comment:
Please move it to the next section (`Insert SCION Workbench Component`) so we have modifications to all files (`html`, `ts`, `scss`) in a single section.

Reply:
Please also update the `how-to-install-workbench.md` file accordingly.